data/campaigns/Liberty/scenarios/08_Glory.cfg

```diff
@@ -169,7 +169,7 @@
             [/avoid]
         [/ai]
         [ai]
-            turns=7-FOREVER
+            turns=7-99999
```

Comment:
Verified with grep on the data directory.

[object]duration has a manifest constant 'forever' which may be the source of the confusion.

The macro is defined as 999 in core/macros/scenario-utils and the only two references are those I changed, here.

BTW INFINITY is also defined as 999, and the only use for that is once in LoW macros. That usage does have {} so should be no problem.

Should I change it back, corrected, to {FOREVER}?